src/components/fields/text-field/types.ts

```diff
@@ -5,6 +5,7 @@ import { IBaseFieldSchema } from "../types";
 type TCustomOptions = {
 	preventCopyAndPaste?: boolean | undefined;
 	preventDragAndDrop?: boolean | undefined;
+	uppercase?: boolean | undefined;
```

Comment:
may I suggest to offer `textTransform` that can accept `uppercase` or `undefined`? this is just in case there is a need to offer other transformation in the future

Reply:
updated

Reply:
updated and resolving